docs/user-guide/commands.md

```diff
@@ -270,6 +270,30 @@ Flag | Help
 `-f, --force` | Don't ask for confirmation.
 `--help` | Show this message and exit.
 
+## `rename`
+
+```bash
+Usage:  watson rename [OPTIONS]
+```
+
+Rename a project or tag.
+
+Example:
+
+
+    $ watson rename project read-python-intro learn-python
+    Renamed project "read-python-intro" to "learn-python"
+    $ watson rename tag company-meeting meeting
+    Renamed tag "company-meeting" to "meeting"
+
+### Options
+
+Flag | Help
+-----|-----
+`-p, --project TEXT...` | Rename project
+`-T, --tag TEXT...` | Rename tag
```

Comment:
These options are deprecated. Did you update the doc via `make docs` (see: https://tailordev.github.io/Watson/contributing/hack/)?